code-monkey/src/tools/__init__.py

```diff
@@ -0,0 +1,38 @@
+from .ask_user_tool import AskUserTool
```

Comment:
Centralized re-exporting is not generally a good idea. I'm not a fan :upside_down_face:

Reply:
really.  can you say why?